lite/examples/image_segmentation/android/app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,6 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="org.tensorflow.lite.examples.imagesegmentation">
 
-  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="29"/>
```

Comment:
Why remove this?

Reply:
Because it is redundant, Gradle will overwrite it with the values specified in the [build script](https://github.com/tensorflow/examples/blob/master/lite/examples/image_segmentation/android/app/build.gradle#L10) (and generate if none exists).